guides/common/modules/proc_enabling-pull-based-transport.adoc

```diff
@@ -2,6 +2,7 @@
 = Enabling Pull-based Transport
 
 .Procedure
+<<<<<<< HEAD
```

Comment:
This looks like you did not properly resolve git merge conflicts. there are a lot of conflict markers in your latest push.

Reply:
@maximiliankolb Fixed & ready to merge!